dockstore-integration-testing/src/test/java/io/dockstore/webservice/NotebookIT.java

```diff
@@ -101,6 +112,56 @@ void testDAOs() {
         session.close();
     }
 
+    @Test
+    void testRegisterSimpleNotebook() {
+        CommonTestUtilities.cleanStatePrivate2(SUPPORT, false, testingPostgres);
+        io.dockstore.openapi.client.ApiClient openApiClient = getOpenAPIWebClient(BasicIT.USER_2_USERNAME, testingPostgres);
```

Comment:
This is basically a new class that only uses openapi client packages, may as well import them to simplify the code (since there is no ambiguity)